var/spack/repos/builtin/packages/py-dnspython/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PyDnspython(PythonPackage):
+    """DNS toolkit"""
+
+    homepage = "https://www.dnspython.org"
+    pypi = "dnspython/dnspython-2.2.1.tar.gz"
+
+    version("2.2.1", sha256="0f7569a4a6ff151958b64304071d370daa3243d15941a7beedf0c9fe5105603e")
+
+    depends_on("python@3.7:", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("python@3.6:", type=("build", "run"))
```
Or just remove it, Spack no longer supports older Python anyway